scripts/onboarding/mdm-alerts/private-preview-onboard-MSI.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+#
```

Comment:
I thought Private preivew scripts where going to be in the /scripts/privatepreview folder. What's the reason to keep this here?

Reply:
> I thought Private preivew scripts where going to be in the /scripts/privatepreview folder. What's the reason to keep this here?

> good catch. let me take care of this.

Reply:
> > I thought Private preivew scripts where going to be in the /scripts/privatepreview folder. What's the reason to keep this here?
> 
> > good catch. let me take care of this.

 Done.